src/index.html

```diff
@@ -9,6 +9,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo"><img src="images/logo.png" alt=""></a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="li">
+            <a href="#" class="link is-active">apple</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__ist">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">apple</a>
```
Найчастіше тобі буде зустрічатись саме така структура. Назва класу `li` - не дуже хороша